### PR TITLE
PMM-9913 Added temp cleanup on agent start/stop

### PR DESCRIPTION
**Pull request by tshcherban**
PMM-9913

Build: SUBMODULES-2586

### CODE REVIEW

File: agent/commands/run.go [hunk 1]
```diff
@@ -55,14 +58,33 @@ func Run() {
 		config.ConfigureLogger(cfg)
 		l.Debugf("Loaded configuration: %+v", cfg)
 
+		cleanupTmp(cfg.Paths.TempDir, l)
+
 		run(ctx, cfg, configFilepath)
 
+		cleanupTmp(cfg.Paths.TempDir, l)
```
Comment (BupycHuk) [inline agent/commands/run.go:65]: don't we remove files on agent stop? I think we should do it on agent stop as well.

Comment (tshcherban) [reply agent/commands/run.go:65]: its done after `run` completes

Comment (tshcherban) [reply agent/commands/run.go:66]: added clear on agent stop

**Review by YaroslavPodorvanov — APPROVED** [review]
LGTM

**Pull request merged**